recipe/meta.yaml

```diff
@@ -25,19 +23,20 @@ requirements:
     - pkg-config
     - make  # [unix]
   host:
-    - coin-or-utils
-    - coin-or-osi
-    - coin-or-clp
-    - libblas
-    - libcblas
-    - liblapack
+    - coin-or-utils 2.11.6
+    - coin-or-osi 0.108.7
+    - coin-or-clp 1.17.7
     - zlib
     - bzip2
-    - ncurses
-    - readline
+    - ncurses 6.2
+    - readline 8.1
```

Comment:
I didn't find mentions of ncurses and readline https://github.com/coin-or/Cgl/blob/releases/0.60.5/.coin-or/config.yml. They are required implicitly?